### ISSUE TITLE
Cannot install latest version (26.0.0) of the adapter using Galleon

### ISSUE THREAD

**Issue by peterfabbro** [keycloak/keycloak#33941]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

adapter/jee

### Describe the bug

Trying to install both Wildfly (33.0.2) and Keycloak SAML adapter (26.0.0) using Galleon 6.0.3. The following commands have been used (the third Galleon call installs the adapter):

```
java -jar "%PATH_GALLEON%\galleon-cli.jar" install wildfly:current#33.0.2.Final --dir=%WF_HOME% --layers=ee-core-profile-server,jaxrs-server,datasources-web-server,deployment-scanner,embedded-activemq,ejb,webservices,undertow-https
java -jar "%PATH_GALLEON%\galleon-cli.jar" install org.wildfly:wildfly-datasources-galleon-pack:9.0.0.Final --dir=%WF_HOME% --layers=postgresql-driver

java -jar "%PATH_GALLEON%\galleon-cli.jar" install org.keycloak:keycloak-saml-adapter-galleon-pack:25.0.0 --dir=%WF_HOME% --layers=keycloak-client-saml-ejb
```

The adapter is successfully installed, when installing various previous versions (including 25.0.6), but installing the latest version does not insert it into the WF server (nothing in the standalone.xml and no modules present). No error is thrown during installation:

```
java -jar "%PATH_GALLEON%\galleon-cli.jar" install org.keycloak:keycloak-saml-adapter-galleon-pack:26.0.0 --dir=%WF_HOME% --layers=keycloak-client-saml-ejb
Generating configuration standalone/standalone.xmlWARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.wildfly.extension.elytron.SSLDefinitions (jar:file:/C:/Users/peter/AppData/Local/Temp/1aa92f6d-53c3-470a-9a62-99de1370e8d9/staged/modules/system/layers/base/org/wildfly/extension/elytron/main/wildfly-elytron-integration-25.0.2.Final.jar!/) to method com.sun.net.ssl.internal.ssl.Provider.isFIPS()
WARNING: Please consider reporting this to the maintainers of org.wildfly.extension.elytron.SSLDefinitions
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

```

### Version

26.0.0

### Regression

- [X] The issue is a regression

### Expected behavior

The adapter installs normally and includes the extension and other elements into the standalone.xml

### Actual behavior

Nothing inserted in the modules subfolder of the server and no extension present in the standalone.xml

### How to Reproduce?

Script commands present in the description.

**Comment by mposolda** [comment]
Triaging this as a blocker until we figure what causes this.

**Comment by peterfabbro** [comment]
Hi @pskopek 

thank you very much for your check and speedy answer.

After trying to run your script as-is and getting to the same problem, I also tried running my script and yours on an actual Ubuntu machine **and there it works normally**. The problem as it seems is therefore present running the commands on Windows (11) and also inside an Ubuntu command line for Windows (WSL).

As a side note, since I have been now able to use the latest version of Keycloak saml adapter, I have also checked the problem https://github.com/keycloak/keycloak/issues/33944 and it is still present.

Thanks again for your checks.
